File: PopularShows/app/src/main/java/com/lgcode/popularshows/network/services/TVShowService.java
package com.lgcode.popularshows.network.services;

import com.lgcode.popularshows.network.response.TVShowListResultResponse;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;
import retrofit2.http.Query;

/**
 * Created by leojg on 11/6/16.
 */
public interface TVShowService {

    @GET("tv/top_rated")
    Call<TVShowListResultResponse> getTVShows(@Query("page") int page);

    @GET("tv/{id}/similar")
    Call<TVShowListResultResponse> getSimilarTVShows(@Path("id") int id, @Query("page") int page);

}

File: KtPopularShows/app/src/main/java/example/lgcode/ktpopularshows/network/TVShowService.kt
package example.lgcode.ktpopularshows.network

import example.lgcode.ktpopularshows.domain.TVShowListResultResponse
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Path
import retrofit2.http.Query

interface TVShowService {

    @GET("tv/top_rated")
    fun getTVShows(@Query("page") page: Int): Call<TVShowListResultResponse>

    @GET("tv/{id}/similar")
    fun getSimilarTVShows(@Path("id") id: Int, @Query("page") page: Int): Call<TVShowListResultResponse>

}